code/game/objects/items/weapons/tools/mods/mod_types.dm

```diff
@@ -403,8 +403,9 @@
 	I.prefix = "silenced"
 
 /obj/item/weapon/tool_upgrade/augment/ai_tool
-	name = "ai tool"
-	desc = "A tool mod from OneStar is deemed heretic by the clan of Technomancers. It enhances the tool by the micro-ai at cost of increased power usage."
+	name = "Nanointegrated AI"
+	desc = "A forgotten One Star tech. This module must be slapped hard enough on a surface of any tool, and you can enjoy right away the benefits it provides. \
```

Comment:
"Composed of forgotten One Star tech. This module is universally compatible with any tool, due to its unique installation method."

Reply:
"A forgotten One Star tech. Due to its unique installation method of \"slapping it hard enough onto anything should do the trick \", it is highly sought after."